### ISSUE TITLE
Add aria-labels to <section>s

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#841]
Part of #820 

Blacklight is using `<section>` tags to define major page areas. On the search results pages, it would be useful for these sections to have aria-labels, to make them more accessible to screen readers. (Landmarks without labels don't show up in landmark lists on at least some SRs.)

- [x] `<section id="content"` add `aria-label="search results"`
- [x] `<section id="sidebar"` add `aria-label="limit your search"`

It's redundant, but gives a more complete picture of the page in the landmarks list.

**Comment by jkeck** [comment]
Just as a heads up, this will be a change upstream in Blacklight.
